src/vendor.ts

```diff
@@ -1,4 +1,9 @@
-import 'angular2/bundles/angular2-polyfills';
+import 'es6-promise';
+import 'es6-shim';
+import 'reflect-metadata';
+import 'zone.js/lib/browser/zone-microtask';
```

Comment:
we are importing from source as we're meant to with webpack for authored application code. Since this is a polyfill we could and probably should use `noParse` or we could import from source to avoid having to include `noParse`